

kernel/pagg.c:260: warning: `task_exited' might be used uninitialized in this function

Signed-off-by: Andrew Morton <akpm@osdl.org>
---

 25-akpm/kernel/pagg.c |    2 +-
 1 files changed, 1 insertion(+), 1 deletion(-)

diff -puN kernel/pagg.c~process-aggregates-warning-fix kernel/pagg.c
--- 25/kernel/pagg.c~process-aggregates-warning-fix	2004-07-28 00:10:14.935047360 -0700
+++ 25-akpm/kernel/pagg.c	2004-07-28 00:10:14.939046752 -0700
@@ -257,7 +257,7 @@ pagg_hook_register(struct pagg_hook *pag
 	/* Now we can call the initializer function (if present) for each task */
 	if (pagg_hook_new->init != NULL) {
 		int init_result = 0;
-		int task_exited;
+		int task_exited = 0;
 
 		/* Because of internal race conditions we can't gaurantee
 		 * getting every task in just one pass so we just keep going
_
